src/components/LifecycleMethodsChart.tsx

```diff
@@ -0,0 +1,198 @@
+import React, { useState } from 'react';
+import Link from '@docusaurus/Link';
```

Comment:
We need to use Docusaurus' `Link` component for any links we do within a React component. This lets us take advantage of some pre-fetching stuff that the component does automatically, and it also ensures that we can utilize Docusaurus' version-aware relative link resolution.

In particular, if we do something like this:

```jsx
<Link to="reactive-data#watch-decorator">@Watch()</Link>
```

That will render as a link to `/docs/v2/reactive-data#watch-decorator`, `/docs/reactive-data#watch-decorator`, or `/docs/next/reactive-data#watch-decorator`, depending on which version of the page we're currently on.

Reply:
See here for more info: https://docusaurus.io/docs/docusaurus-core#link